src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,18 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bladeSize;
+
     public Bulldozer() {
+
```

Comment:
use constructor-based initialization
```suggestion
    public Bulldozer(int bladeSize) {
        this.bladeSize = bladeSize;
```